src/accessors/IHttp.ts

```diff
@@ -1,5 +1,7 @@
 import { IPersistence } from './IPersistence';
 import { IRead } from './IRead';
+//import HTTP_STATUS_CODES from 'http-status-enum';
```

Comment:
Status codes constants would be comfortable here - else, everybody has to create this dependency on his own